submissions/anishchenko/html-css-popup/index.html

```diff
@@ -0,0 +1,157 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/normalize.min.css">
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="shortcut icon" href="img/icons/favicon.png" type="image/x-icon">
+    <title>Hooli Popup</title>
+  </head>
+  <body>
+    <header class="header">
+      <div class="container">
+        <nav class="navigation">
+          <ul class="navigation__list">
+            <li class="navigation__item">
+              <a class="navigation__link" href="#">mail</a>
+            </li>
+            <li class="navigation__item">
+              <a class="navigation__link" href="#">images</a>
+            </li>
+
+            <li class=" navigation__item navigation__item--popup">
+              <div class="popup">
+                <input class="popup__input" name="input-popup" id="popup__input" type="checkbox" >
+                <label class="popup__label" for="popup__input"></label>
+                <ul class="popup__list">
+                  <li class="popup__item">
+                    <a class="popup__link" href="#">
+                      <figure class="popup__figure">
+                        <img class="popup__img" src="img/icons/calendar.png" alt="calendar" width="40" height="30">
```

Comment:
For icons it's better set an empty alt attribute, read more [here](https://accessibility.psu.edu/images/imageshtml/#empty)